server/statshouse/statshouse-client.cpp

```diff
@@ -0,0 +1,69 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2021 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "server/statshouse/statshouse-client.h"
+
+#include <fcntl.h>
+#include <unistd.h>
+
+#include "common/resolver.h"
+#include "net/net-connections.h"
+#include "runtime/critical_section.h"
+#include "server/server-log.h"
+#include "common/tl/constants/statshouse.h"
+#include "common/tl/methods/string.h"
```

Comment:
If includes are sorted — at least "common" are grouped — it looks prettier)

Reply:
done